Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/features/settings/ui/SettingsAdapter.java

```diff
@@ -448,6 +501,19 @@ else if (mClickedItem instanceof FloatSliderSetting)
 
       closeDialog();
     }
+    else if (mClickedItem instanceof InputStringSetting)
+    {
+      InputStringSetting scSetting = (InputStringSetting) mClickedItem;
+      String value = scSetting.getSelectedValue(getSettings());
+      if (!scSetting.getSelectedValue(getSettings()).equals(""))
```

Comment:
This doesn't seem right. All the other cases are comparing the old value to the new value, not comparing the old value to the empty string. Actually, is this even fetching the new value at all...?

Reply:
Changed